spec/controllers/api/v1/bikes_controller_spec.rb

```diff
@@ -151,7 +150,7 @@
           frame_size_unit: nil,
           rear_gear_type_slug: rear_gear_type.slug,
           front_gear_type_slug: front_gear_type.slug,
-          handlebar_type_slug: handlebar_type.slug,
+          handlebar_type_slug: "other",
```

Comment:
Could you make this test something other than `other`? Or the other one, but that does seem to be testing that it fuzzy finds.

Not really that big of a deal, just seems like an easy way to add some extra coverage.

Reply:
Sure thing sounds good 👍